src/razor/src/blazorDebug/blazorDebugConfigurationProvider.ts

```diff
@@ -180,28 +181,33 @@ export class BlazorDebugConfigurationProvider implements vscode.DebugConfigurati
 
     public static async determineBrowserType(): Promise<string | undefined> {
         // There was no browser specified by the user, so we will do some auto-detection to find a browser,
-        // favoring chrome if multiple valid options are installed.
-        const chromeBrowserFinder = new ChromeBrowserFinder(process.env, promises, null);
-        const chromeInstallations = await chromeBrowserFinder.findAll();
-        if (chromeInstallations.length > 0) {
+        // favoring Edge if multiple valid options are installed.
```

Comment:
Thought we should re-order browser priority so that we favor Edge over Chrome (since we're Microsoft 😄). Let me know if anyone has other thoughts.